core/src/main/java/feast/core/model/Field.java

```diff
@@ -47,6 +48,10 @@
   @Column(name = "project")
   private String project;
 
+  // Labels that this field belongs to
+  @Column(name = "labels", columnDefinition = "text")
+  private String labels;
```

Comment:
Hmm so I guess we're going to want to move this to `Entity` in #655… this was mentioned before but I guess I wasn't understanding the implications until I saw #655 reduced from #612.

Creates a bit of conundrum for us since I thought the labels feature could be a straightforward backport (internal, assuming Feast isn't going to backport features). That's not going to be the case if this has to wait on #655 and substantial SQL schema migrations are going to be required for it.